rest/OrderService.js

```diff
@@ -23,7 +26,7 @@ class OrderService {
         return await this.#orderPersistence.getById(orderId);
     }
 
-    async getOrderByProductId(productId) {
+    async getOrdersByProductId(productId) {
```

Comment:
Do we need this here too?

If this method is here that means we plan on making and endpoint for it, which we don't